tests/components/roborock/test_binary_sensor.py

```diff
@@ -9,9 +9,12 @@ async def test_binary_sensors(
     hass: HomeAssistant, setup_entry: MockConfigEntry
 ) -> None:
     """Test binary sensors and check test values are correctly set."""
-    assert len(hass.states.async_all("binary_sensor")) == 4
+    assert len(hass.states.async_all("binary_sensor")) == 6
```

Comment:
Why does this jump 2? Do you have more than one device reporting the new sensor type?

Reply:
Yes - there are two test devices